paper-api-generator/src/main/java/io/papermc/generator/Generators.java

```diff
@@ -36,6 +37,7 @@ public interface Generators {
         simpleKey("DamageTypeKeys", DamageType.class, Registries.DAMAGE_TYPE, RegistryKey.DAMAGE_TYPE, true),
         simpleKey("WolfVariantKeys", Wolf.Variant.class, Registries.WOLF_VARIANT, RegistryKey.WOLF_VARIANT, true),
         simpleKey("ItemTypeKeys", ItemType.class, Registries.ITEM, RegistryKey.ITEM, false),
+        simpleKey("PaintingVariantKeys", Art.class, Registries.PAINTING_VARIANT, RegistryKey.PAINTING_VARIANT, true),
```

Comment:
Can you add missing api like banner patterns, jukebox song etc?

Reply:
just gonna hold off on jukebox songs. We have to update the "naming" because there are 3 pure-number songs, like 11, 13, 5. Need to add logic to convert those to the spelled out versions for field names.